server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java

```diff
@@ -664,6 +668,7 @@ public void enterGroupByFillClause(SqlBaseParser.GroupByFillClauseContext ctx) {
     super.enterGroupByFillClause(ctx);
     queryOp.setGroupBy(true);
     queryOp.setFill(true);
+    queryOp.setGroupByTime(true);
```

Comment:
since there is more than one group by, it's better to use full name.   GroupByLevel and GroupByTime